content/docs/user-guide/gems/reference/aws/aws-core/cdk-application.md

```diff
@@ -5,21 +5,28 @@ weight: 200
 toc: true
 ---
 
-The [Cloud Development Kit](https://docs.aws.amazon.com/cdk/latest/guide/home.html) (CDK) is a software development framework from AWS for defining cloud infrastructure for your project and provisioning it through **AWS CloudFormation**.
+The [Cloud Development Kit](https://docs.aws.amazon.com/cdk/v2/guide/home.html) (CDK) is a software development framework from AWS for defining cloud infrastructure for your project and provisioning it through **AWS CloudFormation**.
```

Comment:
similar to above, prefer full official name and acronym ("AWS Cloud..." / "(AWS CDK)")

Reply:
Fixed